src/test/java/com/epam/tc/hw5/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.tc.hw5.pages;
+
+
+import com.epam.tc.hw5.pages.sections.LogArea;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class DifferentElementsPage extends BasePage {
+
+    public LogArea logArea;
+
+    public DifferentElementsPage(WebDriver webDriver) {
+        super(webDriver);
+        this.logArea = new LogArea(webDriver);
+        this.url = "https://jdi-testing.github.io/jdi-light/different-elements.html";
+    }
+
+    //CheckBoxes
+    @FindBy(css = "div>label.label-checkbox:nth-child(1)>input")
+    public WebElement waterCheckbox;
+    @FindBy(css = "div>label.label-checkbox:nth-child(3)>input")
+    public WebElement windCheckbox;
+
+    //RadioButtons
```

Comment:
try to avoid one line comments, var names are enough

Reply:
fixed